src/btree/bt_vrfy.c

```diff
@@ -261,9 +261,8 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
             WT_ERR(__wt_btree_tree_open(session, root_addr, root_addr_size));
 
             if (WT_VRFY_DUMP(vs))
-                WT_ERR(__wt_msg(session, "Root: %s %s",
```

Comment:
On [this](https://github.com/wiredtiger/wiredtiger/pull/10127/files#diff-02355c543d2d71bdba732d3e6d8ccc31183d15edcd3455ebf3b2a471b6f6c793R252) line. Perhaps this change would improve readability and parsebility.
`WT_ERR(__wt_msg(session, "%s, checkpoint: %s", name, ckpt->name));`

Reply:
ty for the suggestion d1f62736d98dead640b595af67e2dea7aad7a8a2